### DISCUSSION TITLE
Showcases of pages made with VitePress?

### DISCUSSION THREAD

**Discussion by Phhofm** [vuejs/vitepress#1810] [Show and tell]
Is there a place where I can see websites made with VitePress? Would be interested just in seeing different use cases, specifically if people also have used it for like a simple landing page or something similiar, so more like a normal simple website theme rather than documentation theme.

I can also show page I have been making with VitePress, it is running under [https://phhofm.github.io/upscale/](https://phhofm.github.io/upscale/). It is to compare upscaling outputs, have a look at the 'Favorites' Page or the 'Multiple Models' page for example. 

I have to say I had never used VuePress. I had just looked at both (I had been using VueJS and was looking for a static site generator that could be quickly/simply used) and VitePress seemed more interesting to me so I thought I'd try it out. The documentation-layout was perfect for my use case. I simply made markdown pages and integrated a vue component for comparing image upscale outputs, where the image sources come from the github folders where I first traverse the folder and generate a json file with nodejs for all the filenames and links in it first before doing a vitepress build, and I trigger a force rerender on the vue image slider component if another model is selected (for example on the Favorites page, the image controls are meant to be used with a mouse / not mobile friendly) so I can just put a new output file in the folder and make a commit on the main branch and with github actions and everything it will update the examples on the 'Multiple Models' Page this way.

Just thought I'd share of what I have been creating with VitePress. And would also be cool to see  what others have created with VitePress.

(also not sure why vitepress is not listed under https://vue-community.org/guide/ecosystem/static-site-generators.html#existing-solutions for example. Because it is not stable?)

**Comment by kiaking** [comment]
VueTelescope has VitePress tag 👀 
https://www.vuetelescope.com/explore?framework.slug=vitepress&_sort=lastDetectedAt:desc

It could be nice to have Showcase in VitePress docs too when we get to the Stable release 👍

**Comment by alexandrehtrb** [comment]
+1, this should be on vitepress.dev site
